polynote-kernel/src/main/scala/polynote/kernel/logging/package.scala

```diff
@@ -110,21 +115,27 @@ package object logging {
           }
         }.ignore
 
+        private def printWithPrefixSync(prefix: String, indent: String, msg: String)(implicit location: Location): Unit = {
+          val lines = new StringOps(msg).lines
+          val firstLine = lines.next()
```

Comment:
If `msg` is empty this can throw, meaning that a call to `warnSync` could as well. Of course this is not called, and we would have to call `warnSync("")` which we'd probably never do. 

But seeing as we are being extra paranoid and that the `*Sync` calls are meant to be run outside of an effect...

Reply:
`new StringOps("").lines.next` throws??? The empty string is the first line! That makes no sense. 😠 will fix